public/app/features/dashboard/components/DashboardRow/DashboardRow.tsx

```diff
@@ -4,29 +4,36 @@ import { Icon } from '@grafana/ui';
 import { PanelModel } from '../../state/PanelModel';
 import { DashboardModel } from '../../state/DashboardModel';
 import appEvents from 'app/core/app_events';
-import { CoreEvents } from 'app/types';
 import { RowOptionsButton } from '../RowOptions/RowOptionsButton';
 import { getTemplateSrv } from '@grafana/runtime';
-import { ShowConfirmModalEvent } from '../../../../types/events';
+import { RefreshEvent, ShowConfirmModalEvent } from '../../../../types/events';
+import { Unsubscribable } from 'rxjs';
 
 export interface DashboardRowProps {
   panel: PanelModel;
   dashboard: DashboardModel;
 }
 
 export class DashboardRow extends React.Component<DashboardRowProps, any> {
+  sub?: Unsubscribable;
   constructor(props: DashboardRowProps) {
     super(props);
 
     this.state = {
       collapsed: this.props.panel.collapsed,
     };
 
-    this.props.dashboard.on(CoreEvents.templateVariableValueUpdated, this.onVariableUpdated);
+    this.props.dashboard.on(RefreshEvent, this.onVariableUpdated);
```

Comment:
Hmm, do we need this and the line on componentDidMount?

Reply:
oh...my bad.
Would fix up right away